sv/git.go

```diff
@@ -144,7 +144,7 @@ func (g GitImpl) Tag(version semver.Version) (string, error) {
 
 // Tags list repository tags.
 func (g GitImpl) Tags() ([]GitTag, error) {
-	cmd := exec.Command("git", "for-each-ref", "--sort", "creatordate", "--format", "%(creatordate:iso8601)#%(refname:short)", "refs/tags")
+	cmd := exec.Command("git", "for-each-ref", "--sort", "creatordate", "--format", "%(creatordate:iso8601)#%(refname:short)", "refs/tags/" + g.tagCfg.Filter)
```

Comment:
If the tag selector is empty (which it is by default) it works as before. Do you mean that?

Reply:
no, to change the function to receive a filter.. the way it's now, might be misleading, the godocs tells that it return every tag, and it's doing a filter using a value from the config..

eg:
```go
// Tags list repository tags according with the filter, return all if filter is empty.
func (g GitImpl) Tags(filter string) ([]GitTag, error) {
}
`

Reply:
hum.. maybe right now no, but i will probably refactor it in the future since started to bother me..

btw, this filter is only used on the display? or it's to ignore some tags for each command? if so, the filter will need to be added on `func (GitImpl) LastTag() string` too.. 

sorry that i didn't see this before..

Reply:
No problem 😄 I added this to the LastTag as well. It should be to only select certain tags for each command